src/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_suspended_frames.py

```diff
@@ -200,7 +200,7 @@ def get_children_variables(self, fmt=None, scope=None):
 
         return children_variables
 
-    def change_variable(self, name, value, py_db, fmt=None):
+    def change_variable(self, name, value, py_db, fmt=None, scope: Union[ScopeRequest, None]=None):
```

Comment:
Can you use `Optional[Foo]` instead of `Union[Foo, None]`? https://docs.python.org/3/library/typing.html#typing.Optional

Reply:
Yeah that works too. I'll change it. THere was already another spot that did that.